amida-tech/orange-api
# adding side effects to journal entries as their own fields

This PR adds two fields to journal entries to support the side effects feature on the front end.
The two fields are `sideEffect` (string) and `sideEffectSeverity` (number) and are now accepted when creating entries and returned when fetching entries.

To test:
using postman, POST journal entries with  the above two fields, and then GET them back. Ensure it behaves how you'd expect.
Use the orange-ignite app on the branch of the same name to create and view side effects.
